### PR TITLE
Fix bug with <Image> and SSR for configured domains

### PR DESCRIPTION
**Pull request by mabadir**
Closes: #2440

### What are the changes and their implications?

Use `next/image` instead of `@blitz/core/image` when exporting `Image`

## Bug Checklist

- [x] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

### CODE REVIEW

File: nextjs/packages/next/client/image.tsx [hunk 1]
```diff
@@ -241,7 +241,7 @@ function removePlaceholder(
   }
 }
 
-export default function Image({
```
Comment (flybayer) [inline nextjs/packages/next/client/image.tsx:244]: You need to add `export default Image` to keep previous behavior too

**Review by flybayer — COMMENTED** [review]
Also you can now remove the `core/image` entrypoint. To do this, remove the packages/core/image directory and the image files from packages/core/package.json

**Comment by flybayer** [comment]
@mabadir I'd like to get this in the next release tomorrow if possible

**Comment by mabadir** [comment]
So @flybayer, just wondering, are those test cases still relevant:
https://github.com/blitz-js/blitz/tree/canary/test/integration/export-image-default

**Comment by mabadir** [comment]
Now that core/image is removed from blitz. Do we need to export next/image instead? 🤔

**Comment by flybayer** [comment]
@mabadir I just pushed some fixes.

1. Yeah we still needed to export Image stuff from `blitz` for case where babel doesn't rewrite. I debugged this by running `blitz build` inside the misc integration app.
2. I created a new `integration/images` app for your new ssr test because that addition to misc caused `blitz export` to fail since it doesn't work with images. I debugged this by running `blitz export` inside misc integration app.

**Pull request merged**